### ISSUE TITLE
when evaluating

### ISSUE THREAD

**Issue by taku80125** [traveller59/second.pytorch#318]
python ./pytorch/train.py evaluate --config_path=./configs/all.fhd.config --model_dir=/mnt/iso/traveller/second.pytorch/second/ --measure_time=True --batch_size=1
[  41 1280 1056]
feature_map_size [1, 160, 132]
remain number of infos: 3769
Generate output labels...
cudahash: Completely failed to buildit/s][00:02>05:59]   
cudahash: Needed 2 attempts to build, you can ignore this message.
cudahash: Needed 2 attempts to build, you can ignore this message.
[100.0%][===================>][13.76it/s][04:53>00:00]   
generate label finished(12.80/s). start eval:
avg example to torch time: 2.051 ms
avg prep time: 10.489 ms
avg voxel_feature_extractor time = 0.569 ms
avg middle forward time = 35.752 ms
avg rpn forward time = 4.848 ms
avg predict time = 23.928 ms
/home/iso/anaconda3/lib/python3.7/site-packages/numba/compiler.py:602: NumbaPerformanceWarning: 
The keyword argument 'parallel=True' was specified but no transformation for parallel execution was possible.

To find out why, try turning on parallel diagnostics, see http://numba.pydata.org/numba-doc/latest/user/parallel.html#diagnostics for help.

File "utils/eval.py", line 129:
@numba.jit(nopython=True, parallel=True)
def box3d_overlap_kernel(boxes,
^

  self.func_ir.loc))
Evaluation official
Car AP(Average Precision)@0.70, 0.70, 0.70:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00
Car AP(Average Precision)@0.70, 0.50, 0.50:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00
Cyclist AP(Average Precision)@0.50, 0.50, 0.50:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00
Cyclist AP(Average Precision)@0.50, 0.25, 0.25:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00
Pedestrian AP(Average Precision)@0.50, 0.50, 0.50:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00
Pedestrian AP(Average Precision)@0.50, 0.25, 0.25:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00
Van AP(Average Precision)@0.70, 0.70, 0.70:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00
Van AP(Average Precision)@0.70, 0.50, 0.50:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00

Evaluation coco
Car coco AP@0.50:0.05:0.95:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00
Cyclist coco AP@0.25:0.05:0.70:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00
Pedestrian coco AP@0.25:0.05:0.70:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00
Van coco AP@0.50:0.05:0.95:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00

Why all AP are zero??

**Comment by taku80125** [comment]
sorry, this is my false.

**Comment by zfyong** [comment]
I met this problem again, how to solve?

**Comment by Shaoqing26** [comment]
> I met this problem again, how to solve?

I met this problem，Do u solved it？

**Comment by Shaoqing26** [comment]
> I met this problem again, how to solve?

I solved this problem,I forget change the  model file path befor eval.

**Comment by zmlll** [comment]
> > I met this problem again, how to solve?
> 
> I solved this problem,I forget change the model file path befor eval.

i met the same problem  ,which file i should change?

**Comment by AN-Ahmed** [comment]
I'm facing the same issue, how did you solve it?
